src/test/suite/extensionFunctions.test.ts

```diff
@@ -79,31 +79,37 @@ suite('Extension Functions Test Suite', () => {
 	test('try parsing didact url with extension path - extension id in url', async function() {
 		checkCanParseDidactUriForPath(
 			'vscode://redhat.vscode-didact?extension=redhat.vscode-didact/examples/requirements.example.didact.md', 
-			'vscode-didact/examples/requirements.example.didact.md');
+			'vscode-didact/examples/requirements.example.didact.md',
+			'vscode-didact-release/examples/requirements.example.didact.md'); // jenkins
```

Comment:
I don't understand which value is checked and why i tis different?

it sounds like vscode-didact-release is the name of the job. Why is it related to the test?

@bfitzpat Can you clarify what is the root issue please?